### PR TITLE
feat(sku): add new commponent and styles

### PR DESCRIPTION
**Pull request by pmareke**
EX-4517

[Designs](https://www.sketch.com/s/ad08dbbd-2b80-4792-80bb-aff1e1de8ec2/a/zx30ZEa)

image

### CODE REVIEW

File: src/components/predictive-layer.vue [hunk 1]
```diff
@@ -14,8 +14,22 @@
           : 'x-row x-row--gap-06 x-row--align-start'
       "
     >
+      <IdentifierResults
+        #default="{ identifierResult }"
+        :animation="suggestionsAnimation"
+        class="x-row__item x-row__item--span-12 x-list x-list--gap-04"
```
Comment (javieri-empathy) [inline src/components/predictive-layer.vue:20]: The gap should be 8px, `x-list--gap-03`, like in the `QuerySuggestions`.

image

File: src/components/predictive-layer.vue [hunk 2]
```diff
@@ -14,8 +14,22 @@
...
+      >
+        <BaseResultLink #default="{ result }" :result="identifierResult" class="x-suggestion">
+          <BarCodeIcon />
+          <IdentifierResult class="x-text x-text--bold" :result="result" />
```
Comment (javieri-empathy) [inline src/components/predictive-layer.vue:24]: I would limit the results to a maximum of 5, like in the query suggestions.

File: src/components/predictive-layer.vue [hunk 3]
```diff
@@ -14,8 +14,22 @@
           : 'x-row x-row--gap-06 x-row--align-start'
```
Comment (javieri-empathy) [inline src/components/predictive-layer.vue:14]: Not for this PR, but I would try to improve the readability of this template using grouped ifs, and extracting some components. It is starting to be a mess.

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: src/components/predictive-layer.vue (hunk 1 above)
Comment (pmareke) [reply src/components/predictive-layer.vue:20]: image

I see 16px

File: src/components/predictive-layer.vue (hunk 2 above)
Comment (pmareke) [reply src/components/predictive-layer.vue:24]: Me too but @tajespasarela prefers to keep like it is right now.

File: src/components/predictive-layer.vue (hunk 1 above)
Comment (pmareke) [reply src/components/predictive-layer.vue:20]: And 12px 😆 

image

Comment (tajespasarela) [reply src/components/predictive-layer.vue:20]: javi is Right. As I always say, in these cases the designs are illustrative, but we have to keep the consistency in the setup.

so change to `gap--03`

File: src/components/predictive-layer.vue (hunk 2 above)
Comment (tajespasarela) [reply src/components/predictive-layer.vue:24]: let's leave this way at the moment. The SKU search is a special case, to find a result or a set of them. If we "cut" that list, maybe the searched result is out of that.

We can tweak it in each customer setup, but I would leave as it is by now.

File: src/components/predictive-layer.vue (hunk 3 above)
Comment (tajespasarela) [reply src/components/predictive-layer.vue:14]: Right, not in this PR

**Review by javieri-empathy — APPROVED** [review]

**Review by tajespasarela — APPROVED** [review]

**Pull request merged**